source/standards/testing-services.html.md.erb

```diff
@@ -0,0 +1,69 @@
+---
+title: Testing Services
+expires: 2018-01-10
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+## Why and when to test
+
+You should carry out [penetration testing](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing) and vulnerability assessments to ensure your service is secure, according to the [Service Manual](https://www.gov.uk/service-manual):
+
+> "Vulnerability assessments help you find potential weaknesses in your service.
+> Penetration tests proactively attack your systems to find weaknesses and help you understand how easy they are to exploit."
+
+At a minimum you should test twice a year at six month intervals, your tests should consist of:
+
+* one external test by an [approved third party](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing#working-with-third-parties) which is classed as a major test
+* one internal test which is classed as a minor test
+
+In addition to routine testing every 6 months, you should also test following significant changes to your infrastructure or network, for example when you:
+
+* change a cloud service provider
+* change or upgrade an operating system
+* apply security updates or patches
```

Comment:
all out infrastructure should be applying unattended security updates, considering how frequent they are I don't think it's a good indicator that a penetration test is required: https://usn.ubuntu.com/usn/ gives an indicator of how often we apply system patches to packages on ubuntu.

I'd suggest including something about significant changes to firewall configuration / ip whitelists. Internal tests have proved a good way to validate these changes on GOV.UK Pay.

Reply:
Done